WordPress/Classes/Categories/SVProgressHUD+Dismiss.m

```diff
@@ -0,0 +1,52 @@
+#import "SVProgressHUD+Dismiss.h"
+
+@implementation SVProgressHUD (Dismiss)
+
++ (void)showDismissibleErrorWithStatus:(NSString*)status
+{
+    [SVProgressHUD registerForHUDNotifications];
+    [SVProgressHUD setDefaultMaskType:SVProgressHUDMaskTypeClear];
+    [SVProgressHUD showErrorWithStatus:status];
+}
+
++ (void)showDismissibleSuccessWithStatus:(NSString*)status
+{
+    [SVProgressHUD registerForHUDNotifications];
+    [SVProgressHUD setDefaultMaskType:SVProgressHUDMaskTypeClear];
+    [SVProgressHUD showSuccessWithStatus:status];
+}
```

Comment:
Each time this helper gets called, it'll get registered for a couple notifications. Problem:

1. `SVProgressHUD`'s `showDismissableXWithStatus` gets called twice (too close timming)
2. `handleHUDTappedNotification` is expected to be called twice, for a single event
3. `handleHUDDisappearedNotification` would also be called twice, but since it'll remove the hook, will only get hit once.

Suggestion would be to either prevent this scenario from happening (`SVProgressHUD.isVisible`?), or to make sure there's a single NSNotification hook. Say...

```
- (void)showDismissableSuccessWithStatus:...
{
     //...
     [SVProgressHUD unregisterForHUDNotifications];
     [SVProgressHUD registerForHUDNotifications];    
     //...
}
```

Not happy code, though. I need an aspirin!  Nothing is expected to break, though. So let's call this hyper low priority!